src/app/component/shared/items/item/item.component.ts

```diff
@@ -47,7 +47,7 @@ import { StrongRoute } from "src/app/interfaces/strongRoute";
       </span>
     </div>
   `,
-  changeDetection: ChangeDetectionStrategy.OnPush
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
I'd leave a comment explaining why we can use this strategy (immutable objects I presume?)

Reply:
Tbh, i should convert this to take immutable objects only (although this isn't the reason for the change detection strategy). We can set this change detection strategy because its a pure component (only thing that changes is the input so it doesn't need to do change detection). Leaving a comment would be a bit overkill

Reply:
Well that why is exactly the context I was missing... hence the need.